tests/integration/test_postprocessing.py

```diff
@@ -293,3 +298,13 @@ def check_json_data(fieldmaps: list, op: str, value: str):
             data = json.load(f)
             assert rel_ops[op](data["IntendedFor"], value)
             f.close
+
+
+def validateBIDS(bids_root_dir: str):
```

Comment:
so far `bids-validator` wasn't a dependency of xnat-tools. This makes it one. How are you envisioning that?